src/GitVersion.Core/VersionCalculation/VersionCalculators/NextVersionCalculator.cs

```diff
@@ -200,20 +201,47 @@ IEnumerable<NextVersion> GetNextVersionsInternal()
         var incrementedVersion = GetIncrementedVersion(effectiveConfiguration, baseVersion, label);
         if (incrementedVersion.IsMatchForBranchSpecificLabel(label))
         {
-            result = CreateNextVersion(baseVersion, incrementedVersion, effectiveConfiguration);
+            result = new NextVersion(incrementedVersion, baseVersion, effectiveConfiguration);
```

Comment:
Can you also change the calls to the `CreateNextVersion` in this class with `new NextVersion` and get rid of the `CreateNextVersion` method? And then we are good to merge

Reply:
Done